ExampleMod/Content/Items/Accessories/WaspNest.cs

```diff
@@ -11,7 +11,8 @@ namespace ExampleMod.Content.Items.Accessories
 	public class WaspNest : ModItem
 	{
 		// Only gets run once per type
-		public override void Load() {
+		public override void Load(Mod mod) {
+			base.Load(mod);
 			IL.Terraria.Player.beeType += HookBeeType;
```

Comment:
This just feels kinda good

Reply:
Does every modder have to do this in every Load hook? I mean I guess it's fine, but we gotta make sure to document that, we haven't /really/ had the modders use proper inheritance calls yet

Reply:
Yes, but only if they want to put their IL hooks there. The use cases for overriding are limited to modders who will know C# anyway

Reply:
Base calls definitely have to go away. 

> It's a noob trap, and makes it impossible to not call base method in a class that derives from something that derives from a ModX.